devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1161,14 +1425,39 @@ void convertVectorFromDegreesToRadians(iDynTree::VectorDynSize & vector)
 bool WholeBodyDynamicsDevice::readFTSensors(bool verbose)
 {
     bool FTSensorsReadCorrectly = true;
+    bool TempSensorReadCorrectly = true;
+    double timeFTStamp=yarp::os::Time::now();
+    bool readTemperatureSensorThisTime=false;
+    yarp::dev::MAS_status    sensorStatus;
     for(size_t ft=0; ft < estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE); ft++ )
     {
         iDynTree::Wrench bufWrench;
         int ftRetVal = ftSensors[ft]->read(ftMeasurement);
+//TODO: change 0.55 for a variable. 0.55 was related to the fact that temperature sensors are updated once every second
+        if (timeFTStamp-prevFTTempTimeStamp>0.55){
```

Comment:
magic number! Please address this TODO in this PR.

Reply:
I will address it then. I guess it will be another configuration parameter.

Reply:
created variable checkTemperatureEvery_seconds with default value of 0.55